app/models/slug_migration_publisher.rb

```diff
@@ -0,0 +1,22 @@
+require "gds_api/publishing_api_v2"
+
+class SlugMigrationPublisher
+  def process(slug_migration)
+    data = {
+      content_id: slug_migration.content_id,
+      format: "redirect",
+      publishing_app: "service-manual-publisher",
+      base_path: slug_migration.slug,
+      redirects: [
+        {
+          path: slug_migration.slug,
+          type: "exact",
+          destination: slug_migration.guide.slug,
```

Comment:
hmm, not related to this PR, but I'm surprised that destination is not a content-id